### PR TITLE
Implement validation for commitmemt of material transaction

### PR DESCRIPTION
**Pull request by Yamaguchi**
resolve https://github.com/chaintope/tapyrus-api/issues/351

### CODE REVIEW

File: frontend/src/app/transaction/transaction.page.ts [hunk 1]
```diff
@@ -70,4 +70,6 @@ export class TransactionPage implements OnInit {
     });
     return await modal.present();
   }
+
+  goToValidation() {}
```
Comment (azuchi) [inline frontend/src/app/transaction/transaction.page.ts:74]: Is this change necessary?

File: backend/actions/validate.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    script.substring(0, 2) == '6a' && // OP_RETURN
+    script.substring(2, 4) == '26' && // size(38 bytes)
+    script.substring(4, 8) == '5450' && // marker
+    script.substring(8, 10) == '02' && // version
+    script.substring(10, 12) == '22' && // payload size(22 bytes)
+    (script.substring(12, 14) == '01' ||
+      script.substring(12, 14) == '02' ||
+      script.substring(12, 14) == '03') && // operation
+    script.substring(14) == commitment
+  ); //commitment
```
Comment (azuchi) [inline backend/actions/validate.js:30]: Commitments must be calculated from each material and checked to match payload commitments.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: frontend/src/app/transaction/transaction.page.ts (hunk 1 above)
Comment (Yamaguchi) [reply frontend/src/app/transaction/transaction.page.ts]: 不要でした。
削除します(127673159cf523766bf84f4320da8eb863cfb61c)

File: backend/actions/validate.js (hunk 2 above)
Comment (Yamaguchi) [reply backend/actions/validate.js:31]: 875285639192fb82dd6ce28b16e1c560c63f8007

File: frontend/src/app/tracking_validation/tracking_validation.page.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  </div>
+  <div class="ion-padding ion-margin">
+    <div class="description">
+      It can be verified that the content of the
+      <strong>opened value</strong> matches the commitment recorded in the
+      blockchain. Opened Value is a value given to disclose the content of the
+      commitments included in a tracking transaction. For more information on
+      opened value, please see to the
+      <a
+        href="https://github.com/chaintope/glueby-contract-tracking/blob/main/doc/v2spec.md#54-%E9%96%8B%E7%A4%BA%E5%80%A4"
```
Comment (azuchi) [inline frontend/src/app/tracking_validation/tracking_validation.page.html:13]:
```suggestion
        href="https://doc.api.tapyrus.chaintope.com/#tag/material_tracking"
```

**Review by azuchi — CHANGES_REQUESTED** [review]
